guides/common/modules/proc_enabling-the-red-hat-flatpak-remote.adoc

```diff
@@ -0,0 +1,74 @@
+[id="enabling-the-red-hat-flatpak-remote_{context}"]
+= Enabling the Red Hat Flatpak remote
+
+This Hammer CLI procedure enables and manages Flatpak repositories.
+
+.CLI procedure
+. Enable a Flatpak remote on the {ProjectServer} using either standalone {EL} systems or {ProjectName}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ hammer flatpak-remote create --name=$name --url=$url
```

Comment:
We need to provide an organization identifier here..Similar to other hammer commands that are scoped by organization.